src/components/Dashboard/Form/Button/Button.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import styles from "./Button.scss";
+import PropTypes from "prop-types";
+
+const Button = props => (
+  <div>
+    <button type={props.type} className={styles.Button} onClick={props.onClick}>
+      {props.label}
```

Comment:
Could we remove `props.label` here and replace it with `props.children`? It will allow us to use this component more like native buttons, eg. `<Button onClick={doSomething}`>I want my cookies!</Button>`. All usages of this Button component will need to be replaced with new notation